### PR TITLE
Add undefined log.file.* fields breaking tests for filestream inputs

### PR DESCRIPTION
**Pull request by bhapas**
## What does this PR do?

Issue described in https://github.com/elastic/integrations/issues/7687

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates https://github.com/elastic/integrations/issues/7687

### CODE REVIEW

File: packages/oracle/data_stream/database_audit/fields/agent.yml [hunk 1]
```diff
@@ -202,3 +202,24 @@
 - name: log.offset
   type: long
   description: Log offset
+- name: log.file
```
Comment (muthu-mps) [inline packages/oracle/data_stream/database_audit/fields/agent.yml:205]: As a part of this change, Is it okay to move the existing field `log.file.path` from ecs.yml. This way we can have all the `log.file.*` fields in a single place.

Comment (bhapas) [reply packages/oracle/data_stream/database_audit/fields/agent.yml:205]: But ,`log.file.path` is still an ECS field https://www.elastic.co/guide/en/ecs/current/ecs-log.html#field-log-file-path

**Comment by muthu-mps** [comment]
@bhapas - Did you consider updating the sample-events json file with the new data with the log.file.* fields ?

**Comment by bhapas** [comment]
@muthu-mps Updated after rerunning the system tests

**Review by muthu-mps — APPROVED** [review]
LGTM!

**Pull request merged**